### PR TITLE
MOSIP-31977: Merging 1.2.0.1 to develop

### PR DESCRIPTION
**Pull request by HimajaDhanyamraju2**

### CODE REVIEW

**Review by vishwa-vyom — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Websub
-[![Ballerina Build and push](https://github.com/mosip/websub/actions/workflows/push_trigger.yml/badge.svg?branch=release-1.2.0)](https://github.com/mosip/websub/actions/workflows/push_trigger.yml)
-[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?branch=release-1.2.0&project=mosip_websub&metric=alert_status)](https://sonarcloud.io/dashboard?branch=release-1.2.0&id=mosip_websub)
+[![Ballerina Build and push](https://github.com/mosip/websub/actions/workflows/push_trigger.yml/badge.svg?branch=release-1.2.0.1)](https://github.com/mosip/websub/actions/workflows/push_trigger.yml)
```
Comment (ckm007) [inline README.md:2]: please update the branch name to develop

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?branch=release-1.2.0.1&project=mosip_websub&metric=alert_status)](https://sonarcloud.io/dashboard?branch=release-1.2.0.1&id=mosip_websub)
```
Comment (ckm007) [inline README.md:3]: update branch as develop

File: README.md (hunk 1 above)
Comment (HimajaDhanyamraju2) [reply README.md]: Done, pls check now.

File: README.md (hunk 2 above)
Comment (HimajaDhanyamraju2) [reply README.md]: Done, pls check now.

**Review by ckm007 — APPROVED** [review]

**Pull request merged**